recipes/libredwg/meta.yaml

```
@@ -0,0 +1,64 @@
+{% set name = "libredwg" %}
+{% set version = "0.8.2327" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/LibreDWG/libredwg/releases/download/{{ version }}/libredwg-{{ version }}.tar.gz
+    sha256: 259f204b896f374a7324916cc4ad28936c542ae5c9fc34bd39a02fa77dcb27fc
+  - url: https://raw.githubusercontent.com/LibreDWG/libredwg/{{ version }}/build-aux/git-version-gen
+    sha256: 129de06e36ec87882ff12a44baec1cabb1cc0fa034db4495e7d33632f4abcb01
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler("c") }}  # [unix]
+    - {{ compiler('m2w64_c') }}  # [win]
+    - swig
+    - m2-autoconf  # [win]
+    - m2-make  # [win]
+    - m2-libtool  # [win]
+    - m2-m4  # [win]
+    - m2-automake-wrapper  # [win]
+    - m2-texinfo  # [win]
+    - posix  # [win]
+    - autoconf  # [unix]
+    - automake  # [unix]
+    - libtool  # [unix]
+    - m4  # [unix]
+    - texinfo  # [unix]
+  host:
+    - python
+    - libxml2
+    - perl
+  run:
+    - python
+    - libxml2
+
+test:
+  imports:
+    - LibreDWG
+
+about:
+  home: https://www.gnu.org/software/libredwg/
+  doc_url: https://www.gnu.org/software/libredwg/
+  dev_url: http://savannah.gnu.org/projects/libredwg/
+  license: GPL3
```

SPDX identifiers are preferred:
```suggestion
  license: GPL-3.0-or-later
```

---

:+1: